app/src/main/kotlin/at/bitfire/davdroid/resource/LocalCalendarStore.kt

```diff
@@ -97,6 +97,14 @@ class LocalCalendarStore @Inject constructor(
         return values
     }
 
+    override fun updateAccount(
+        provider: ContentProviderClient,
+        oldAccount: Account,
+        newAccount: Account
+    ) {
+        throw NotImplementedError()
+    }
```

Comment:
I just tried again and it seemed that it actually works? 😄 I updated `Calendars.CONTENT_URI.asSyncAdapter(oldAccount)` with `Calendars.ACCOUNT_NAME` set to the new account name, and the calendars were moved together with the events.

Please check + if it works for you too, adapt accordingly.